Content/posts/5-3-released.md

```diff
@@ -0,0 +1,71 @@
+---
+date: 2020-11-10 0:39
+description: The first stable toolchain release of SwifWasm!
+---
+
+# The first stable toolchain release of SwifWasm!
+
+## Overview 
+
+This is the first public release of SwiftWasm toolchain, available as a signed .pkg installer for macOS. Also via swiftenv-compatible archives and Docker for Intel-based Ubuntu 18.04 and 20.04.
+The focus is providing the basic and stable Swift features on WebAssembly platform.
+
+
+This release includes the Swift for WebAssembly compiler, the standard and core libraries, JavaScript interoperation library, UI library, build tool and CI supports.
+
+
+## The standard library and core libraries
+
+The Swift standard library is fully available on WebAssembly platform.
+
+The standard library now depends on WASI, which is a modular system interface for WebAssembly, to build the stdlib upon [wasi-libc](https://github.com/WebAssembly/wasi-libc).
+However, we are going to make the WASI dependency optional.
+
+
+### Foundation / XCTest
+
+Foundation and XCTest are also available on WebAssembly, but in a limited capacity.
+
+You can learn more from our documentation:
+
+- https://book.swiftwasm.org/getting-started/foundation.html
+- https://book.swiftwasm.org/getting-started/testing.html
```

Comment:
Please refer to our [Foundation](https://book.swiftwasm.org/getting-started/foundation.html) and [XCTest](https://book.swiftwasm.org/getting-started/testing.html) guides for more details.
```